.tekton/pipeline-service-ckcp-testing.yaml

```diff
@@ -0,0 +1,295 @@
+                  sleep 60
+                  wait_period=$((wait_period + 60))
+                  echo "Waited $wait_period seconds..."
+                done
+
+                echo "Hypershit is ready, the following is the cluser kubeconfig"
+                kubectl get secret -n clusters "$(params.cluster-name)-admin-kubeconfig" -o json | jq -r '.data.kubeconfig'  | base64 -d | tee $(workspaces.output.path)/kubeconfig
+      - name: fetch-repository
+        runAfter:
+          - "deploy-cluster"
+        taskRef:
+          name: git-clone
+          kind: ClusterTask
+        workspaces:
+          - name: output
+            workspace: source
+        params:
+          - name: url
+            value: $(params.repo_url)
+          - name: revision
+            value: $(params.revision)
+      - name: ckcp-test
+        runAfter:
+          - "fetch-repository"
+        workspaces:
+          - name: source
+            workspace: source
+          - name: kubeconfig-dir
+            workspace: shared-workspace
+        taskSpec:
+          workspaces:
+            - name: source
+            - name: kubeconfig-dir
+          kind: ClusterTask
+          steps:
+            - name: create-podman-container
+              image: quay.io/xinjiang/kubectl:1.25
+              resources:
+                requests:
+                  memory: 500Mi
+                  cpu: 300m
+              script: |
+                #!/usr/bin/env bash
+                export KUBECONFIG=$(workspaces.kubeconfig-dir.path)/kubeconfig
+
+                kubectl -n openshift-config  get secret/pull-secret -o yaml | sed 's/openshift-config/default/' | sed 's/\.dockerconfigjson/auth.json/' | grep -v type: | kubectl apply -f -
+                kubectl -n default apply -f - <<EOF
+                apiVersion: v1
+                kind: Pod
+                metadata:
+                  name: podman
+                spec:
+                  containers:
+                  - name: podman
+                    image: registry.redhat.io/rhel8/podman:latest
+                    resources:
+                      requests:
+                        ephemeral-storage: "2Gi"
+                      limits:
+                        ephemeral-storage: "2Gi"
+                    command:
+                    - /usr/sbin/init
+                    securityContext:
+                      privileged: true
+                    volumeMounts:
+                    - name: pull-secret
+                      mountPath: "/run/user/0/containers/"
+                      readOnly: true
+                  volumes:
+                  - name: pull-secret
+                    secret:
+                      secretName: pull-secret
+                EOF
+                kubectl -n default wait pod/podman --for=condition=Ready --timeout=90s
+                kubectl -n default describe pod/podman
+            - name: install-softwares-to-podman-container
+              image: quay.io/xinjiang/kubectl:1.25
```

Comment:
Change this to the official image. If this is a custom image you've built, then please upload this to quay.io/redhat-pipeline-service organization.

Reply:
fixed, same as above